### PR TITLE
Id ward RTD Module: initial release

### PR DESCRIPTION
**Pull request by kyrylenko**
## Type of change

- [x] Feature

## Description of change

Adding a new RTD module for [Id Ward](https://id-ward.com).

- contact email of the adapter’s maintainer pavlo@id-ward.com

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/pull/3578

### CODE REVIEW

**Comment by kyrylenko** [comment]
@gwhigs may I ask you when you would be able to review it? Thanks in advance!

File: test/spec/modules/idWardRtdProvider_spec.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  };
+
+  const onDone = function() { return true };
+
+  const cmoduleConfig = {
+    'name': 'idWard',
+    'waitForIt': true,
+    'params': {
+      'cohortStorageKey': 'cohort_ids',
+      'segtax': 503
```
[the remark this answers is not in the document]
Comment (patmmccann) [reply test/spec/modules/idWardRtdProvider_spec.js:22]: please see https://github.com/InteractiveAdvertisingBureau/openrtb/blob/master/extensions/community_extensions/segtax.md

File: modules/idWardRtdProvider.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  try {
+    return JSON.parse(data);
+  } catch (err) {
+    logError(`idWardRtdProvider: failed to parse json:`, data);
+    return null;
+  }
+}
+
+/**
+  * Real-time data retrieval from Audigent
```
Comment (patmmccann) [inline modules/idWardRtdProvider.js:60]: is this your stuff @anthonylauzon

File: integrationExamples/gpt/idward_segments_example.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
+<html>
+<head>
+    <script async src="../../build/dev/prebid.js"></script>
+    <script async src="https://www.googletagservices.com/tag/js/gpt.js"></script>
```
Comment (patmmccann) [inline integrationExamples/gpt/idward_segments_example.html:4]: this is out of date

File: modules/idWardRtdProvider.js [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+const SUBMODULE_NAME = 'idWard';
+
+export const storage = getStorageManager(null, SUBMODULE_NAME);
+
+/**
+  * Lazy merge objects.
+  * @param {String} target
+  * @param {String} source
+  */
+function mergeLazy(target, source) {
```
Comment (patmmccann) [inline modules/idWardRtdProvider.js:23]: did you see the utils function that does this?

File: modules/idWardRtdProvider.js (hunk 2 above)
Comment (kyrylenko) [reply modules/idWardRtdProvider.js]: corrected

File: integrationExamples/gpt/idward_segments_example.html (hunk 3 above)
Comment (kyrylenko) [reply integrationExamples/gpt/idward_segments_example.html]: used `securepubads.g.doubleclick.net/tag/js/gpt.js` instead

File: modules/idWardRtdProvider.js (hunk 4 above)
Comment (kyrylenko) [reply modules/idWardRtdProvider.js]: used `mergeDeep`

**Review by patmmccann — APPROVED** [review]

**Pull request merged**